win32ss/gdi/eng/pandisp.c

```diff
@@ -0,0 +1,484 @@
+/*
+* PROJECT:     ReactOS Win32k subsystem
+* LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+* FILE:        win32ss/gdi/eng/pandisp.c
+* PURPOSE:     Pan-Display driver
+* PROGRAMMERS: Copyright 2022 Hervé Poussineau (hpoussin@reactos.org)
+*/
```

Comment:
```suggestion
/*
 * PROJECT:     ReactOS Win32k subsystem
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Pan-Display driver
 * COPYRIGHT:   Copyright 2022 Hervé Poussineau <hpoussin@reactos.org>
 */
```
Per https://reactos.org/wiki/Coding_Style